src/test_l1_hdmi_cec_driver.c

```diff
@@ -2231,16 +2238,33 @@ static UT_test_suite_t *pSuite_panel = NULL;
  */
 int test_hdmicec_hal_l1_register( void )
 {
-
+    ut_kvp_status_t status = UT_KVP_STATUS_SUCCESS;
+    char    deviceType[UT_KVP_MAX_ELEMENT_SIZE];
+    // Reading Extended enum support form profile file
+    extendedEnumsSupported = UT_KVP_PROFILE_GET_BOOL("hdmicec/features/extendedEnumsSupported");
+    // Getting device type from profile.
+    UT_KVP_PROFILE_GET_STRING("hdmicec/yype",deviceType);
```

Comment:
`hdmicec/yype"` -> typo?

Reply:
updated. As per the conversation below, the check has been moved to main.c